ordsys/asgi.py

```diff
@@ -8,11 +8,17 @@
 """
```

Comment:
https://github.com/UTNkar/ordsys/blob/d15eea92266a7e08d8360f18ce4b8d41b8b0667a/ordsys/asgi.py#L1-L8

The documentation used in the above comment might no longer relevant, consider referring to either [Django](https://docs.djangoproject.com/en/4.0/howto/deployment/asgi/) or [channels](https://channels.readthedocs.io/en/stable/topics/routing.html)

Reply:
I changed so that it referes to channels